docs/admins.rst

```diff
@@ -229,6 +229,15 @@ default_signing_intent
   Name of the default signing intent to be used when one is not provided
   in ``container.yaml``.
 
+If HTTPS is enabled for the access to YUM repositories hosted in ODCS,
```

Comment:
Not just ODCS, we have implemented it for any yum repo, even those passed by users via parameter.

Reply:
Well, if so, this is not the right section to add the new config description. This section is dedicated for odcs. Which section is proper to have a general description for the `builder_ca_bundle`?

Reply:
Just make it as a new section see `Limiting image size` for example

Reply:
It can be section `Custom CA bundle for yum repos` or so